drivers/bluetooth/hci/CMakeLists.txt

```diff
@@ -32,7 +32,8 @@ zephyr_library_sources_ifdef(CONFIG_BT_CYW208XX  hci_ifx_cyw208xx.c)
 
 zephyr_library_sources_ifdef(CONFIG_BT_STM32_IPM   ipm_stm32wb.c)
 zephyr_library_sources_ifdef(CONFIG_BT_STM32WBA    hci_stm32wba.c)
-if(CONFIG_DT_HAS_ST_HCI_STM32WBA_ENABLED)
+zephyr_library_sources_ifdef(CONFIG_BT_STM32WB0    hci_stm32wb0.c)
+if((CONFIG_DT_HAS_ST_HCI_STM32WBA_ENABLED) OR (CONFIG_DT_HAS_ST_HCI_STM32WB0_ENABLED))
```

Comment:
We should soon start making vendor CMakeLists and Kconfig file, these files start getting crowded now.